modules/developers-guide/pages/java-setup.adoc

```diff
@@ -0,0 +1,32 @@
+== gRPC Java client setup
+
+=== Prerequisites
+
+* xref:authnz.adoc[Get an application token]
+* Create a table in your keyspace(optional): xref:rest-using.adoc#_create_a_keyspace[(REST)]  xref:graphql-first-using.adoc#_create_a_keyspace[(GraphQL)]
+* xref:start-stargate.adoc[Start Stargate (optional)]
+
+=== Install software
+
+You'll need to install the following software if it is not already installed:
+
+* link:https://docs.oracle.com/javase/8/docs/technotes/guides/install/install_overview.html[Install a Java Development Kit (JDK)]
```

Comment:
@tomekl007 or @mpenick we should be able to run the client with Java 8 or higher, right?

Reply:
```suggestion
* link:https://adoptopenjdk.net/[Install a Java Development Kit (JDK)]
```
OpenJDK for consistency in the other parts of the docs

Reply:
changed. also changed in astra docs